db/revision.go

```diff
@@ -466,6 +466,15 @@ func containsUserSpecialProperties(b Body) bool {
 	return false
 }
 
+func ContainsSpecialProperties(b Body) bool {
```

Comment:
If "Special properties" has a bit of a different meaning in the "strip" functions. Might be worth changing it to make it clear:

```
// stripSpecialProperties returns a copy of the given body with all underscore-prefixed keys removed, except _attachments and _deleted.
// stripAllSpecialProperties returns a copy of the given body with all underscore-prefixed keys removed.
```

```suggestion
func ContainsAnySpecialProperties(b Body) bool {
```